charts/posthog/values.yaml

```diff
@@ -715,12 +715,41 @@ grafana:
     datasources.yaml:
       apiVersion: 1
       datasources:
+
+      # Uncomment if you are running with `prometheus.enabled: true`
```

Comment:
I don't understand, they already are uncommented?

Reply:
My bad, I was thinking to do the other way around but we can probably simply remove the comments